src/utils/bivariate/helpers/converters/axisDTOtoAxis.ts

```diff
@@ -1,11 +1,16 @@
-import { formatBivariateAxisLabel } from '~utils/bivariate/labelFormatters';
+import {
+  formatBivariateAxisLabel,
+  formatCustomLabelForBivariateAxis,
+} from '~utils/bivariate/labelFormatters';
 import type { AxisDTO } from '~core/resources/bivariateStatisticsResource/types';
 import type { Axis } from '~utils/bivariate/types/stat.types';
 
 export function axisDTOtoAxis(dto: AxisDTO): Axis {
   return {
     ...dto,
     id: dto.quotient.join('|'),
-    label: dto.label || formatBivariateAxisLabel(dto.quotients),
+    label: dto.label
+      ? formatCustomLabelForBivariateAxis(dto.label, dto.quotients)
```

Comment:
maybe for consistency name function as `formatBivariateAxisCustomLabel` to make difference more clear?

Reply:
fixed